docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,703 @@
+      source = "redhat-developer/rhoas"
+      version = "0.3.0"
+    }
+  }
+}
+
+provider "rhoas" {
+  #configuration options
+}
+----
+
+. Open your IDE of choice and paste the code into it. You can specify configuration options in the provider settings. For example, you can enter the offline token here using the `offline_token` field. However as noted previously, it is recommended to enter the offline token as an environment variable.
++
+
+. Save the file as a .tf file.
+. Open a terminal and enter the following command in the Terraform directory. This command initializes the working directory containing Terraform configuration files and installs any required plug-ins.
++
+[source,shell]
+----
+$ terraform init
+----
+A success message displays that `Terraform has been successfully initialized!`
+
+[id="con-using-terraform_{context}"]
+=== Using the RHOAS Terraform provider
+
+Resources are the most important element in the Terraform language. Each resource block describes one or more infrastructure objects. Terraform has error messages that help with minor typing errors by suggesting a resource close to what is typed. The next steps show what resources (either for single tasks or a combination of tasks) you can add to your Terraform configuration file to provision your Kafka instance.
+
+[id"proc-creating-kafka-instance-terraform_{context}"]
+[discrete]
+=== Creating a Kafka instance
+
+[role="_abstract"]
+To manage a Kafka instance using Terraform, use the `rhoas_kafka` resource.
+
+.Prerequisites
+
+* You have an offline token that is accessible to Terraform.
+* You have a Red Hat account.
+
+.Procedure
+
+. Open your IDE of choice and enter the example rhoas_kafka resource. This example uses the “instance”  identifier and creates a Kafka instance called `my-instance`. The only required fields are the `name`, `billing_model` and `plan` fields. The following values are set by default when you run `terraform apply`. All other information for the instance is provided by the Kafka APIs.
++
+.. `cloud provider`: A list of available cloud providers can be obtained using `data.rhoas_cloud_providers`.
+.. `region`: A list of available regions can be obtained using
+`data.rhoas_cloud_providers_regions`
+
++
+.Example `rhoas_kafka` resource
+[source,shell]
+----
+$ terraform {
+  required_providers {
+    rhoas = {
+      source  = "registry.terraform.io/redhat-developer/rhoas"
+      version = "0.3.0"
+    }
+  }
+}
+
+provider "rhoas" {}
+
+resource "rhoas_kafka" "my-instance" {
+  name = "my-instance"
+}
+
+output "bootstrap_server_my-instance" {
+  value = rhoas_kafka.my-instance.bootstrap_server_host
+}
+----
+. Open a terminal and initilize Terraform by entering the following command.
++
+[source,shell]
+----
+$ terraform init
+----
+. Apply the changes by entering the following command.
++
+[source,shell]
+----
+$ terraform apply
+----
+. Terraform displays a message that `rhoas_kafka.my-instance` will be created. Enter *yes*.
```

Comment:
The first part of this should be the result of the previous step. Then this might step might read something like the following:

"When you're ready to create your instance, type `Yes`."